### PR TITLE
Labels for dialogs

### PR DESCRIPTION
**Pull request by Lunalo**
@rdstern 

Can you comment about the the text for the label

### CODE REVIEW

File: instat/dlgName.Designer.vb [hunk 1]
```diff
@@ -79,21 +78,40 @@ Partial Class dlgName
...
         Me.ucrReceiverName.TabIndex = 2
         '
+        'lblVariableLabel
+        '
+        Me.lblVariableLabel.AutoSize = True
+        Me.lblVariableLabel.Location = New System.Drawing.Point(255, 141)
+        Me.lblVariableLabel.Name = "lblVariableLabel"
+        Me.lblVariableLabel.Size = New System.Drawing.Size(77, 13)
+        Me.lblVariableLabel.TabIndex = 6
+        Me.lblVariableLabel.Text = "Variable Label:"
```
Comment (dannyparsons) [inline instat/dlgName.Designer.vb]: "Column Label:" or just "Label:" would be fine I think.

**Comment by Lunalo** [comment]
@africanmathsinitiative/developers  This is ready for review

**Comment by dannyparsons** [comment]
Would be good to get this in today as it adds an important new feature. Lily's comments need to be addressed first though.

**Comment by Lunalo** [comment]
@dannyparsons  @lilyclements 

This is ready

**Pull request merged**